src/_locales/sk/messages.json

```
@@ -0,0 +1,338 @@
+    "popup_paste_title": {
+        "message": "Prilepiť odkaz [V]"
+    },
+    "popup_server_title": {
+        "message": "Zmeniť server"
+    },
+    "popup_preferences_title": {
+        "message": "Nastaviť"
+    },
+    "popup_report_title": {
+        "message": "Nahlásiť chybyu / Požiadať o novú funkciu"
+    },
+    "popup_loading_title": {
+        "message": "Pripájam sa..."
+    },
+    "popup_donate_title": {
+        "message": "Podpora"
+    },
+    "popup_rate_title": {
+        "message": "Ohodnoťte vašu skúsenosť"
+    },
+    "popup_previous_title": {
+        "message": "Prejsť na predchádzajúce [PageUp]"
+    },
+    "popup_rewind_title": {
+        "message": "Skok dozadu [R]"
+    },
+    "popup_stop_title": {
+        "message": "Zastaviť [X]"
+    },
+    "popup_pause_title": {
+        "message": "Pauza [Space]"
+    },
+    "popup_play_title": {
+        "message": "Prehrať [Space]"
+    },
+    "popup_forward_title": {
+        "message": "Skok dopredu [F]"
+    },
+    "popup_next_title": {
+        "message": "Prejsť na ďalšie [PageDown]"
+    },
+    "popup_volume_title": {
+        "message": "$VALUE$% [-/+]",
+        "placeholders": {
+            "value": {
+                "content": "$1"
+            }
+        }
+    },
+    "popup_mute_title": {
+        "message": "Stíšiť / Obnoviť hlasitosť [F8]"
+    },
+    "popup_repeat_title": {
+        "message": "Opakovať"
+    },
+    "popup_shuffle_title": {
+        "message": "Náhodne"
+    },
+    "popup_clear_title": {
+        "message": "Vyčistiť"
+    },
+    "popup_contextmenu_title": {
+        "message": "Otvoriť kontextovú ponuku [C]"
+    },
+    "popup_up_title": {
+        "message": "Hore [Arrow Up]"
+    },
+    "popup_info_title": {
+        "message": "Zobraziť informácie [I]"
+    },
+    "popup_left_title": {
+        "message": "Vľavo [Arrow Left]"
+    },
+    "popup_select_title": {
+        "message": "Vybrať [Enter]"
+    },
+    "popup_right_title": {
+        "message": "Vpravo [Arrow Right]"
+    },
+    "popup_back_title": {
+        "message": "Späť [Backspace]"
+    },
+    "popup_down_title": {
+        "message": "Dole [Arrow Down]"
+    },
+    "popup_osd_title": {
+        "message": "Zobraziť OSD [M]"
+    },
+    "popup_home_title": {
+        "message": "Prejsť domov"
+    },
+    "popup_fullscreen_title": {
+        "message": "Vypnúť/zapnúť celú obrazovku [Tab]"
+    },
+    "popup_opensendtext_title": {
+        "message": "Poslať text"
+    },
+    "popup_playerprocessinfo_title": {
+        "message": "Zobraziť informácie o prehrávaní [O]"
+    },
+    "popup_empty_textcontent": {
+        "message": "Playlist prázdny"
+    },
+    "popup_playItem_title": {
+        "message": "Prehrať"
+    },
+    "popup_removeItem_title": {
+        "message": "Odstrániť"
+    },
+    "popup_text_placeholder": {
+        "message": "Text na odoslanie"
+    },
+    "popup_done_textcontent": {
+        "message": "Skončiť po odoslaní"
```

[Kore](https://play.google.com/store/apps/details?id=org.xbmc.kore) use _Po odoslaní ukončiť_ for _Finish after send_.

```suggestion
        "message": "Po odoslaní ukončiť"
```

---

Equivalents (order does not change the meaning)